pages/index.js

```diff
@@ -15,10 +16,22 @@ export default function Index () {
       </div>
     )
   }
+
+  if (!isUserFetched) {
```

Comment:
If the user is never fetched, this will continuously show this loading message. There needs to be some logic for what happens when a user is unable to login or doesn't have a login in the first place.

Reply:
I've fixed this (somewhat) in a recent commit. The issue appears to be that `isLoading` from `userAuth` is is not working: https://github.com/sandrinodimattia/use-auth0-hooks/issues/14

Also, as @binh-dam-ibigroup  mentioned elsewhere, we should probably migrate to https://github.com/auth0/auth0-react during next sprint at some point